src/routes/safe/components/Transactions/TxList/QueueTxList.tsx

```diff
@@ -79,7 +82,13 @@ type QueueTxListProps = {
 
 export const QueueTxList = ({ transactions }: QueueTxListProps): ReactElement => {
   const { txLocation } = useContext(TxLocationContext)
-  const title = txLocation === 'queued.next' ? 'NEXT TRANSACTION' : 'QUEUE'
+  const nonce = useSelector(safeNonceSelector)
+
+  const getTitle = () => {
+    return txLocation === 'queued.next'
+      ? 'NEXT TRANSACTION'
+      : `QUEUE - Transaction with nonce ${nonce} needs to be executed next`
```

Comment:
Maybe "needs to be executed ~next~ first"?

Reply:
up to @tschubotz

Reply:
@tschubotz what do you think about changing "needs to be executed next" to "needs to be executed first"?

Reply:
No strong opinion about this one to be honest. The main reason why I went with `next` is  that the section on top is called `_next_ transaction` (not `first tx`). 

Why would `first` be better?

Reply:
Because `next` in this case is ambiguous:
* it can mean "next after the one you're hovering over" (which is wrong), or
* "next" as in "the next action you do in the app", like "zunächst" in German (which is correct)

Reply:
hm I see your point. 
With `first` though, assuming the `safe nonce=5` and there are 3 txs (`nonce=5,6,7`):
Now hovering over tx with `nonce=7` would say `tx with nonce 5 has to be executed first`. That's true. But after tx with `nonce=5` has been executed it would say now `tx with nonce 6 has to be executed first`. So kind of only half the truth. 
I wouldn't make it more complex though, i.e. e.g. `Txs with nonce 5-6 have to be executed first`. 

So having said that. I'm happy with `first` as well, it seems to be less ambiguous. Let's change it!